app.js

```diff
@@ -9,20 +9,8 @@ if (process.env.LOG_TO_JSON === 'true') {
 }
 
 // For Shunter configuration documentation, refer to: https://shunter.readthedocs.io/en/latest/usage/configuration-reference/
-const app = shunter({
-  path: {
-    themes: __dirname
-  },
-  routes: config.routes,
-  jsonViewParameter: 'json',
-  modules: [config.moduleName],
-  log: logger
-});
+const app = shunter(config.shunter(__dirname, process.env.LOG_TO_JSON));
 
-if (process.env.LOG_TO_JSON === 'true') {
-  app.use(middlewares.morgan);
-}
-
-app.use('/health-check', middlewares.healthCheck);
+middlewares.add(app, process.env.LOG_TO_JSON);
```

Comment:
Rename this method to something like `bootstrap` which more clearly described the behaviour underneath